src/DocumentFormat.OpenXml/GeneratedCode/schemas_openxmlformats_org_wordprocessingml_2006_main.g.cs

```diff
@@ -32168,7 +32168,7 @@ public PrinterSettingsReference PrinterSettingsReference
         }
 
         /// <summary>
-        /// <para>FootnoteColumns.</para>
+        /// <para>FootnoteColumns, this property is only available in Office 2013 and later..</para>
```

Comment:
File name indicates schema from year 2006. Is it correct that this parameter is available only starting in Office 2013?

Reply:
Same issue here; w15:footnoteColumns was added directly to the w:secPr (previous section properties) and this caused it to be added to the existing 2006 named generated file.